Source/Scene/GlobeSurfaceShaderSet.js

```diff
@@ -187,6 +191,12 @@ define([
             vs.sources.push(getPositionMode(sceneMode));
             vs.sources.push(get2DYPositionFraction(useWebMercatorProjection));
 
+            if (receiveShadows) {
+                // TODO : should have a better way of modifying the GlobeVS and GlobeFS shaders...
+                vs.sources[1] = ShadowMapShader.createReceiveShadowsVertexShader(vs.sources[1]);
+                fs.sources[0] = ShadowMapShader.createReceiveShadowsFragmentShader(fs.sources[0], frameState.context);
+            }
```

Comment:
This is fine for now - standard for the globe shader.

Longer-term (which could be soon), we need a better low-level material system (that still allows dynamically recompiling the shader for when the shadow flags are enabled/disabled).  Perhaps we will beef up `ShaderProgram` to know about shadows, and add flags to commands for if they cast/receive shadows.